mule-artifact-it/mule-packager-it/src/test/java/integration/test/mojo/ProcessSourcesMojoTest.java

```diff
@@ -39,11 +43,22 @@
       "/empty-classloader-model-project/target/repository/org/mule/group/mule-plugin-b/1.0.0/classloader-model.json";
   private static final String EXPECTED_CLASSLOADER_MODEL_FILE =
       "/expected-files/expected-classloader-model.json";
+  private static final String EXPECTED_LIGHTWEIGHT_LOCAL_REPOSITORY_CLASSLOADER_MODEL_FILE =
+      "/expected-files/expected-lightweight-local-repository-classloader-model.json";
   private static final String EXPECTED_MULE_PLUGIN_A_CLASSLOADER_MODEL_FILE =
       "/expected-files/expected-mule-plugin-a-classloader-model.json";
   private static final String EXPECTED_MULE_PLUGIN_B_CLASSLOADER_MODEL_FILE =
       "/expected-files/expected-mule-plugin-b-classloader-model.json";
 
+  private static final String GENERATED_MULE_PLUGIN_A_LIGHTWEIGHT_LOCAL_REPOSITORY_CLASSLOADER_MODEL_FILE =
```

Comment:
we should test a case where the same plugin is declared with different versions (e.g: in the app and as a transitive of another plugin), and see that the classloader-model.json is correctly located and found